src/store/block.js

```diff
@@ -54,25 +56,30 @@ export default {
       date: el.date,
       harvester: el.signer.address.address
     })),
-    getPageIndex: util.getPageIndex,
-    getSubscription: util.getSubscription,
-    getLoading: util.getLoading,
-
+    getPageIndex: state => state.pageIndex,
+    getSubscription: state => state.subscription,
+    getLoading: state => state.loading,
     blockInfo: state => state.blockInfo,
     blockTransactionList: state => state.blockTransactionList,
     currentBlockHeight: state => state.currentBlockHeight,
     blockInfoLoading: state => state.blockInfoLoading,
     blockInfoError: state => state.blockInfoError
   },
   mutations: {
-    setLatestList: util.setLatestList,
-    setPageList: util.setPageList,
-    setPageIndex: util.setPageIndex,
-    setSubscription: util.setSubscription,
-    setLoading: util.setLoading,
-    resetPageIndex: util.resetPageIndex,
-    addLatestItem(state, item) {
-      util.addLatestItemByKey(state, item, 'height', 1)
+    setLatestList: (state, list) => { state.latestList = list },
```

Comment:
May lose reactivity. For objects use `Vue.set(state, "latestList", list)` instead please

Reply:
@OlegMakarenko, aren't we only calling this with `commit`, which should ensure reactivity regardless? Vue seems to not require `Vue.set` for mutations in the store, since those are reactive by nature.

Reply:
@Alexhuszagh Sorry, my bad. Yes, in this case you are right. Refresh whole object with new one - it works. There is issue with mutating object properties in Vuex Store